### ISSUE TITLE
Test results on ScanNet

### ISSUE THREAD

**Issue by mrharicot** [alexrich021/3dvnet#8]
Hi,

Thank you for sharing your code!
Could you please also share the meshes generated by your method? similar to what you did for VoRTX ?

Thanks

**Comment by alexrich021** [comment]
Yup I can absolutely do that! It will take me a couple of days to get to.

Thanks for your interest!

-Alex
